README.md

```diff
@@ -32,6 +32,7 @@ metadata:
 - sed
 - jq
 - yq (python-yq from https://github.com/kislyuk/yq#installation, other distributions may not work)
+- skopeo
```

Comment:
well, it's a prereq only for a small subset of targets... Maybe that's OK that Makefile instructs that it's needed and it should not be on a global level?

Reply:
up to you folks, feel free to close it
however, there is a typo in the instructions that should be fixed in any case https://github.com/devfile/devworkspace-operator/pull/591

Reply:
```suggestion
- skopeo (if building the OLM catalogsource)
```